### DISCUSSION TITLE
add fixed section above Current Entities in developer-tools/state

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#8882] [Other feature requests]
or if you would phrase differently, start scrolling the page from Current entities. reason: on a system with more than a few entities we need to scroll down to click that entity and then scroll all the way up again for top section and set the representation.

Would really be convenient if that could be fixed and made more user friendly.
thanks for considering 

<img width="755" alt="Schermafbeelding 2021-04-09 om 17 15 56" src="https://user-images.githubusercontent.com/33354141/114202412-717aa680-9957-11eb-8035-a8bbe0df3015.png">

**Comment by spacegaier** [comment]
I think that would cause problems on smaller screens as potentially the whole screen is covered by the top section, so the user could no longer see and use the entity table.

**Reply by Mariusthvdb** [reply]
maybe a device dependent responsive design could be created for that? 
One could also think of an auto-scroll to the top section upon clicking an entity in the list. Maybe an arrow in or replacing the scroll bar to go up to the top section.. 

I must say, that mobile devices (Phones) tend to be less useful in the developer-state (there is no attributes list anyways) but explicitly the fact we need to scroll manually is a nuisance on mobile, even more than on desktop. (scroll bar is useless on  mobile) and clicking/selecting an entity now does nothing at all, at least not visible...
